### DISCUSSION TITLE
Tabella Stato nelle circolari non visibile

### DISCUSSION THREAD

**Discussion by nicolatrentini** [iisgiua/giuaschool#663] [Richieste di aiuto]
Pubblicando una circolare sul Registro, e andando su un profilo docente nella sezione Bacheca > Circolari escono le tabelle:
Numero, Data, Oggetto ma non esce la tabella Stato.
Invece nella sezione Bacheca > Avvisi esce la tabella Stato.
Come posso risolvere?

**Comment by trinko** [comment]
La colonna STATO è presente solo se la circolare appartiene all'anno scolastico corrente. 
Per le circolari archiviate (anni scolastici precedenti) non viene visualizzato perché non è rilevante.
